src/stylesheets/core/_variables.scss

```diff
@@ -616,9 +628,9 @@ $h3-font-size:        2rem !default;
 $h4-font-size:        1.7rem !default;
 $h5-font-size:        1.5rem !default;
 $h6-font-size:        1.3rem !default;
-$base-line-height:    1.5 !default;
-$heading-line-height: 1.3 !default;
-$lead-line-height:    1.7 !default;
+$base-line-height:    1.5rem !default;
+$heading-line-height: 1.3rem !default;
+$lead-line-height:    1.7rem !default;
```

Comment:
These values should all be unitless to scale with font size. Can you change this back?

Reply:
Oh ha — I knew this looked wrong.